src/js/view/components/nextuptooltip.js

```diff
@@ -74,10 +75,12 @@ define([
                 return;
             }
             dom.addClass(this.container, 'jw-nextup-container-visible');
+            dom.addClass(this._playerElement, 'jw-flag-nextup');
         },
         hide: function() {
             dom.removeClass(this.container, 'jw-nextup-container-visible');
             dom.removeClass(this.container, 'jw-nextup-sticky');
+            dom.removeClass(this._playerElement, 'jw-flag-nextup');
```

Comment:
I don't think this was a good call. When muted, only show the mute button. Nextup is not more important.